lib/rialto/etl/service_client/web_of_science_client.rb

```diff
@@ -56,10 +56,12 @@ def perform_initial_query!
 
         def connect_with_retries(path:)
           response = client.get(path)
+          raise "#{response.reason_phrase}: #{response.status}  (#{response.body})" unless response.success?
           JSON.parse(response.body)
         rescue StandardError => exception
-          logger.warn "Error fetching #{path}: #{exception.message} (#{exception.class})"
-          {}
+          msg = "Error fetching #{path}: #{exception.message} (#{exception.class})"
+          logger.warn msg
+          raise msg
```

Comment:
Does this conflict with what's in the `RetriableConnectionFactory`? https://github.com/sul-dlss/rialto-etl/blob/master/lib/rialto/etl/service_client/retriable_connection_factory.rb#L30-L55 We want to retry the errors and errors conditions that are specified there and not bomb out the whole run.